server/api/label.go

```diff
@@ -72,14 +72,14 @@ func (h *labelsHandler) GetStores(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	stores := cluster.GetStores()
+	stores := cluster.GetMetaStores()
 	storesInfo := &StoresInfo{
 		Stores: make([]*StoreInfo, 0, len(stores)),
 	}
 
 	stores = filter.filter(stores)
 	for _, s := range stores {
-		store, err := cluster.GetStore(s.GetId())
+		store, err := cluster.TryGetStore(s.GetId())
```

Comment:
Because we already have a function named `GetStore`. It doesn't return err. We can merge these two functions into one. But we need to change a lot of code including interfaces. I will raise another PR to do it.